# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/copyvertex.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/renderer/copyvertex.h
--- a/gfx/angle/checkout/src/libANGLE/renderer/copyvertex.h	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/copyvertex.h	Fri Nov 08 17:06:00 2019 +0000
@@ -24,40 +24,34 @@
           size_t inputComponentCount,
           size_t outputComponentCount,
           uint32_t alphaDefaultValueBits>
-inline void CopyNativeVertexData(const uint8_t *input,
+void CopyNativeVertexData(const uint8_t *input, size_t stride, size_t count, uint8_t *output);
+
+template <size_t inputComponentCount, size_t outputComponentCount>
+void Copy8SintTo16SintVertexData(const uint8_t *input,
                                  size_t stride,
                                  size_t count,
                                  uint8_t *output);
 
-template <size_t inputComponentCount, size_t outputComponentCount>
-inline void Copy8SintTo16SintVertexData(const uint8_t *input,
-                                        size_t stride,
-                                        size_t count,
-                                        uint8_t *output);
-
 template <size_t componentCount>
-inline void Copy8SnormTo16SnormVertexData(const uint8_t *input,
-                                          size_t stride,
-                                          size_t count,
-                                          uint8_t *output);
+void Copy8SnormTo16SnormVertexData(const uint8_t *input,
+                                   size_t stride,
+                                   size_t count,
+                                   uint8_t *output);
 
 template <size_t inputComponentCount, size_t outputComponentCount>
-inline void Copy32FixedTo32FVertexData(const uint8_t *input,
-                                       size_t stride,
-                                       size_t count,
-                                       uint8_t *output);
+void Copy32FixedTo32FVertexData(const uint8_t *input, size_t stride, size_t count, uint8_t *output);
 
 template <typename T, size_t inputComponentCount, size_t outputComponentCount, bool normalized>
-inline void CopyTo32FVertexData(const uint8_t *input, size_t stride, size_t count, uint8_t *output);
+void CopyTo32FVertexData(const uint8_t *input, size_t stride, size_t count, uint8_t *output);
 
 template <bool isSigned, bool normalized, bool toFloat>
-inline void CopyXYZ10W2ToXYZW32FVertexData(const uint8_t *input,
-                                           size_t stride,
-                                           size_t count,
-                                           uint8_t *output);
+void CopyXYZ10W2ToXYZW32FVertexData(const uint8_t *input,
+                                    size_t stride,
+                                    size_t count,
+                                    uint8_t *output);
 
 }  // namespace rx
 
-#include "copyvertex.inc"
+#include "copyvertex.inc.h"
 
 #endif  // LIBANGLE_RENDERER_COPYVERTEX_H_